app/res/values/themes.xml

```diff
@@ -73,7 +73,9 @@
 
     <style name="ActionBar.TitleTextStyle">
         <item name="android:textColor">@color/white</item>
-        <item name="android:paddingLeft">8dp</item>
+        <item name="android:paddingLeft">16dp</item>
+        <item name="android:textSize">18sp</item>
+        <item name="android:textStyle">bold</item>

```

Comment:
Are these values coming from figma as well ? I would think we should keep the style same as `TitleTextStyle` as thats standard Android sizes and font styles.